src/libraries/Microsoft.PowerFx.Interpreter/Functions/LibraryMath.cs

```diff
@@ -935,5 +935,79 @@ private static ErrorValue GetDiv0Error(IRContext irContext)
                 Message = "Division by zero"
             });
         }
+
+        private static FormulaValue Dec2Hex(IRContext irContext, NumberValue[] args)
+        {
+            var min_number = -Math.Pow(2, 39);
+            var max_number = Math.Pow(2, 39) - 1;
+
+            var number = Math.Floor(args[0].Value);
+            var places = Math.Floor(args[1].Value);
+
+            if (double.IsInfinity(number) || number < min_number || number > max_number || double.IsInfinity(places))
+            {
+                return CommonErrors.OverflowError(irContext);
```

Comment:
>CommonErrors.OverflowError(irContext) [](http://example.com/codeflow?start=23&length=37)

                return new ErrorValue(irContext, new ExpressionError()
                {
                    Message = $"Places should be between 0 and 10",
                    Span = irContext.SourceContext,
                    Kind = ErrorKind.Numeric
                });